Simplify the k8s docker configuration (#968)
RoadieHQ/marketing-site: content/docs/integrations/kubernetes/index.md

### Step 5: (Optional, for brokered connections) Configure your Broker client

> If you are contacting to your Kubernetes clusters via a brokered connection, you can run a container within your kubernetes cluster as a pod or you can run the container outside.

Reviewer: ```suggestion
If you are contacting to your Kubernetes clusters via a brokered connection, you can run a container within your Kubernetes cluster as a pod or you can run the container outside.
```

> If you are contacting to your Kubernetes clusters via a brokered connection, you can run a container within your kubernetes cluster as a pod or you can run the container outside.

Reviewer: ```suggestion
If you are contacting to your Kubernetes clusters via a brokered connection, you can run a container within your kubernetes cluster as a pod or you can run the container as a standalone service.
```

#### Running a docker conatiner

> You can run the roadie kubernetes broker client, outside of a pod, if you provide it with a service token, ca file and endpoint. As follows, please replace the `$TENANT_NAME` with your tenant name:

Reviewer: ```suggestion
You can run the Roadie Kubernetes broker client, outside of a pod, if you provide it with a service token, ca file and endpoint. As follows, please replace the `$TENANT_NAME` with your tenant name:
```

#### Running in a k8s pod

If you would like to run the broker container within a pod in your cluster, you will need to create a service account for the pod that would allow it to access the resources neccessary to operate the backstage plugin. Here is an example of such a manifest. Please note to replace the "$TENANT_NAME" with your Roadie tenant name.

<details>

<Summary>Show broker-kube-manifest.yaml</Summary>
```yaml
---
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: read-stuff
rules:
  - apiGroups: [""]
    resources:
      - pods
      - configmaps
      - services
      - deployments
      - replicasets
      - horizontalpodautoscalers
      - ingresses
    verbs: 
      - "get"
      - "list"
      - "watch"
  - apiGroups: ["batch"]
    resources:
      - jobs
      - cronjobs
    verbs: 
      - "get"
      - "list"
      - "watch"
  - apiGroups: ["extensions", "apps", "autoscaling", "networking.k8s.io"]

>     resources: ["deployments", "ingresses", "replicasets", "horizontalpodautoscalers"]

Reviewer: We are mixing two different kinds of list formats here and the `resources` block above. Let's use one only
